src/convertToObject.js

```diff
@@ -6,7 +6,29 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const rules = sourceString.split(';');
+
+  rules.forEach((rule) => {
+    if (!rule.trim()) {
+      return;
+    }
+
+    const [property, value] = rule.split(':');
+
+    if (!property || !value) {
```

Comment:
The check for '!property || !value' is redundant because you have already checked for '!rule.trim()' which ensures that both property and value exist if the rule is not empty.

Reply:
Remove it please